compiler/testSuite/05-narrow/13-e.bal

```diff
@@ -3,6 +3,6 @@ public function main() {
     int x = 1;
     if x == 1 {
     }
-    else if x == 1 {  // this should get an error because x should be narrowed to not include `1` so the operand types are disjoint
+    else if x == 1 {  // @error
```

Comment:
Worth preserving the original commit, just move it above.

Reply:
fixed